pkg/executor/sortexec/sort.go

```diff
@@ -160,7 +169,7 @@ func (e *SortExec) Open(ctx context.Context) error {
 		e.diskTracker.AttachTo(e.Ctx().GetSessionVars().StmtCtx.DiskTracker)
 	}
 
-	e.IsUnparallel = true
+	e.IsUnparallel = false
```

Comment:
when will it be true?

Reply:
> when will it be true?

Always be false. We retain this variable, because unparallel sort codes haven't been deleted yet.